tests/Integration/tests/configuration/input/guides.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<guides xmlns="https://www.phpdoc.org/guides"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.phpdoc.org/guides vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd"
```

Comment:
I think this should be a valid, callable URI, no? Because it isn't, even when removing the space in the URI?

Reply:
Never mind, I misread the docs. It's right.